source/slides/12_guest_and_midterm.rst

```diff
@@ -5,6 +5,124 @@ Geoffrey Corey
 
 Guest Speaker
 
+Vagrant
+-------
+
+Vagranftiles are... *ok*
+
+.. code-block:: ruby
+
+  # Single box with default options.
+  #
+  Vagrant::Config.run do |config|
+    config.vm.box = 'precise32'
+    config.vm.box_url = 'http://files.vagrantup.com/precise32.box'
+  end
+
+`Complex`_ Vagrantfiles are not
```

Comment:
You might want to put "not ok" to  clarify what you mean. I had to think about it for a moment.